feature_tests/dart/lib/src/BorrowedFields.g.dart

```diff
@@ -26,17 +26,45 @@ final class BorrowedFields {
     c = Utf8Decoder().convert(underlying.c._pointer.asTypedList(underlying.c._length));
 
   // ignore: unused_element
-  _BorrowedFieldsFfi _pointer(ffi.Allocator temp) {
+  // If this struct contains any slices, their lifetime-edge-relevant objects (typically _FinalizedArenas) will only
```

Comment:
I *kiiinda* want people to be able to understand what the code does when they poke at it but in this case I get it

Reply:
I changed my mind: I think it's useful to have these in the generated code to track things down, especially since which code handles which edges is nontrivial